src/components/SingleRecipePage/SingleRecipePage.tsx

```diff
@@ -114,7 +115,7 @@ const SingleRecipePage = () => {
     recipe.recipeTags.map((tag: RecipeTag) =>
       renderingTags.push(tag.tagName.toLocaleLowerCase())
     );
-    recipe.recipeSpecialDiets.map((diet: string) => {
+    recipe.recipeSpecialDiets.forEach((diet: string) => {
```

Comment:
Since we're not returning anything here (we're pushing to an existing array), we want to use `forEach` instead of `map`

Reply:
thank you!